jvm-libs/generic/extensions/kotlin/src/main/kotlin/build/linea/URIExtensions.kt

```diff
@@ -0,0 +1,16 @@
+package build.linea
+
+import java.net.URI
+
+fun URI.getPortWithSchemeDefaults(): Int {
```

Comment:
Typo, `Scheme` -> `Schema`

Reply:
It was actually on purpose. My initial implementation was `Shema`, but then I saw that Java uses `scheme` and googled and the are synonyms, so I went to the [RFC](https://datatracker.ietf.org/doc/html/rfc3986) to check the official term and it's scheme. So I made the change to [scheme](https://github.com/Consensys/linea-monorepo/pull/230/commits/8a01ae95681ba22288ce9cba6cd36b6dc6ee27cf).

Reply:
Ok, good to know